omod/src/main/webapp/pages/metadata/locations/locationAttributeType.gsp

```diff
@@ -129,7 +130,7 @@
                     label        : ui.message("adminui.maxOccurs"),
                     formFieldName: "maxOccurs",
                     id           : "maxOccurs",
-                    maxLength    : 101,
+                    maxLength    : 9,
```

Comment:
Thanks @CrooksDC   Cant the maximunLength be more than 9 at some scenarios?

Reply:
I guess it could be but if there if the number is bigger than 2,147,483,647(max number an int can be) then there is the UI Framework Error number exception. With limiting it to 9 the biggest number we can have is 999,999,999 which is guarantees that this page won't have the UI Framework Error.